spec/integration/post_show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
```

Comment:
For `User's Posts Index`:

Please add a test that asserts the user can see the user's profile picture.
Please add a test that asserts the user can see the user's username.
Please add a test that asserts the user can see the number of posts the user has written.
Please add a test that asserts the user can see a post's title.
Please add a test that asserts the user can see some of the post's body.
Please add a test that asserts the user can see the first comments on a post.
Please that asserts user can see how many comments a post has.
Please add a test that asserts the user can see how many likes a post has.
Please add a test that asserts users can see a section for pagination if there are more posts than fit on the view.
Please add test that redirects to the post's show page when you click on a post.

Reply:
Thanks for pointing out, I have fixed all of them now 😀